schemas/1.6.0/adaptive-card.json

```diff
@@ -3279,6 +3361,28 @@
           "description": "Label for this input",
           "version": "1.3"
         },
+        "labelPosition": {
+          "$ref": "#/definitions/InputLabelPosition",
+          "description": "Determines the position of the label. It can take 'inline' and 'above' values. By default, the label is placed 'above' when label position is not specified.",
```

Comment:
is the label position implemented for mobile renderers?

Reply:
Indicated that the support is only in the JS SDK.